app.go

```diff
@@ -282,7 +285,9 @@ type PushProvider struct {
 	APNTeamID  string `json:"apn_team_id,omitempty"`
 	APNTopic   string `json:"apn_topic,omitempty"`
 
-	FirebaseCredentials string `json:"firebase_credentials,omitempty"`
+	FirebaseCredentials          string  `json:"firebase_credentials,omitempty"`
+	FirebaseNotificationTemplate string  `json:"firebase_notification_template,omitempty"`
```

Comment:
We should be able to reset this template to default one the same way as FirebaseAPNTemplate